core/src/main/java/com/scalar/db/transaction/consensuscommit/ConsensusCommitUtils.java

```diff
@@ -314,4 +318,38 @@ public static int getNextTxVersion(@Nullable Integer currentTxVersion) {
       return currentTxVersion + 1;
     }
   }
+
+  static void createAfterImageColumnsFromBeforeImage(
```

Comment:
[minor] This method name sounds to create a new collection. Something like `copy` or `extract` might be better?

Reply:
`create` sounds fine to me. (because I proposed it and it's more like a behavior from external.)
I feel `copy` and `extract` are more like explaining the internal behavior.
`extract` also sounds to me like after image is a subset of before image.
We also discussed `build`, but the upper-level method uses `create`, so using a different action name seems to intentionally mean something.

Reply:
Let's discuss this in a separate PR. Thanks!